lib/controllers/.xervo-base-ignore

```diff
@@ -1,8 +1,11 @@
 .git/
 .modulus-base-ignore
+.xervo-base-ignore
 .modulusignore
+.xervoignore
 .meteor/local/
 modulus-auto-zip-*
```

Comment:
Those files may/probably exist on users machines now. In a future release we can deprecate them, and remove after that.